oktoberfest/utils/config.py

```diff
@@ -298,6 +303,17 @@ def nr_ox(self) -> int:
         """Get the maximum number of oxidations allowed on M residues in peptides during spectral library generation."""
         return self.spec_lib_options.get("nrOx", 1)
 
+    ######################################
+    # these are local prediction options #
+    ######################################
+
+    # TODO expand this when transfer learning API is released
+
+    @property
+    def predict_intensity_locally(self) -> bool:
```

Comment:
I am not sure why we need this: If providing a path to the model of dlomix, it would be easier for the user then, if oktoberfest automatically detects this? Is there a reason to have it? If this is true, but you would use a koina model, then you actually have to catch this in the Predictor, which you do. Is it easier to check if the user provides a local path? Or is it difficult to determine if the provided string is a path or a model (then we need to have this option)?

Reply:
In that case, I would also recommend to have this argument within the model section of the config. What do you reckon?

Reply:
fair point, now checking if model=="baseline"/"", is a non-existent Keras file, or an existing path